src/store/config/index.ts

```diff
@@ -18,7 +18,8 @@ export const defaultState = (): ConfigState => {
       locales: [
         { name: 'English', code: 'en' },
         { name: 'Français', code: 'fr' },
-        { name: 'Chinese', code: 'cn' }
+        { name: 'Chinese', code: 'cn' },
+		{ name: 'Nederlands', code: 'nl' }
```

Comment:
I take it that your editor didn't read the `.editorconfig` file, can you check if this is not a tab and/or if you have the correct amount of spaces in there?

Reply:
Fixed

Reply:
Thanks